nextcord/client.py

```diff
@@ -1981,7 +1992,7 @@ def user_command(
         description: :class:`str`
             Description of the command that users will see. If not set, it defaults to the bare minimum Discord allows.
         guild_ids: Iterable[:class:`int`]
-            IDs of :class:`Guild`'s to add this command to. If unset, this will be a global command.
+            IDs of :class:`Guild`'s to add this command to. If unset, it will use the client rollout guild ids.
```

Comment:
This could be made more clear. It currently seems that setting client guild ids will add Guilds **in addition** to the set guild ids, not instead of.

I think it would make sense to use the command's `guild_ids` **instead of** the client `rollout_guild_ids` if they are set, but the way I'm reading the code, it seems like it is not that way right now.

Also, these descriptions should probably still mention that you can make it global by overriding guild_ids with the default or not setting either.